req_test.go

```diff
@@ -0,0 +1,76 @@
+// ⚡️ Fiber is an Express inspired web framework written in Go with ☕️
+// 🤖 Github Repository: https://gitee.com/azhai/fiber-u8l
+// 📌 API Documentation: https://docs.gofiber.io
+
+package fiber
+
+import (
+	"net/http/httptest"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var expectBodies = []string{
+	`{"code":200,"data":{"param":"john"}}`,
+	`{"code":200,"data":{"page":1,"size":20}}`,
+}
+
+func CreateTestGroup() *Group {
+	grp := New().Group("/test")
+	grp.Get("/:param", func(c *Ctx) {
+		param := c.Params("param")
+		data := Map{"param": param}
+		c.JSON(Map{"code": 200, "data": data})
+	})
+	grp.Get("/page/:page/:size", func(c *Ctx) {
+		page, size := 1, 20
+		if pageStr := c.FormValue("page"); pageStr != "" {
+			page, _ = strconv.Atoi(pageStr)
+		}
+		if sizeStr := c.FormValue("size"); sizeStr != "" {
+			size, _ = strconv.Atoi(sizeStr)
+		}
+		data := Map{"page": page, "size": size}
+		c.JSON(Map{"code": 200, "data": data})
+	})
+	return grp
+}
+
+//func CreateSeniorGroup() *Group {
```

Comment:
Why do you comment it?

Reply:
The test is used for another unofficial version  [fiber-u8l](https://gitee.com/azhai/fiber-u8l)
I has add some methods like ParamStr(), FetchInt() in Ctx class.